packages/protobuf/src/private/names.ts

```diff
@@ -246,17 +237,17 @@ const reservedIdent: { [k: string]: boolean } = {
   globalThis: true,
   Uint8Array: true,
   Partial: true,
-};
+} satisfies { [k: string]: boolean };
```

Comment:
`satisfies` is nice, but only available from TS v4.9. Please revert so we retain the option to publish types for older compilers.